python_scripts/02_numerical_pipeline_scaling.py

```diff
@@ -348,14 +348,14 @@
 # ![Cross-validation diagram](../figures/cross_validation_diagram.png)
```

Comment:
We could as well have a colour legend if possible. This would be nice. It could be done in another PR.